test/unit/user_test.rb

```diff
@@ -138,10 +138,6 @@ def setup
     end
   end
 
-  test ".try_to_login if password is empty should return nil" do
```

Comment:
There is a test right below this one, `when password is empty should return nil` that is identical.